### PR TITLE
demo: search engine packages installation

### PR DESCRIPTION
**Pull request by hachreak**
Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

File: invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/requirements-devel.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+-e git+git://github.com/inveniosoftware/invenio-marc21.git#egg=invenio-marc21
+-e git+git://github.com/hachreak/invenio-indexer.git@cli_reindex#egg=invenio-indexer
```
Comment (jirikuncar) [inline invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/requirements-devel.txt]: `s/hachreak/inveniosoftware/`?

File: invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/setup.py [hunk 2]
```diff
@@ -21,10 +22,17 @@
...
     entry_points={
         'console_scripts': [
-            '{{ cookiecutter.package_name }} = {{ cookiecutter.package_name }}.cli:cli'
+            '{{ cookiecutter.package_name }} = '
+            '{{ cookiecutter.package_name }}.cli:cli'
         ],
     },
     install_requires=[
-        'invenio-base>=1.0.0a3',
+        #  'invenio-base>=1.0.0a3',
```
Comment (jirikuncar) [inline invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/setup.py]: you still need `invenio-base`

File: invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/requirements-devel.txt (hunk 1 above)
Comment (hachreak) [inline invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/requirements-devel.txt]: They are temporary workaround to be able to create the `invenio` demosite. 
After the PR'll be merged, I'll restore.

Comment (jirikuncar) [inline invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/requirements-devel.txt]: then please add WIP to PR title!

File: invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/{{ cookiecutter.package_name }}/templates/{{ cookiecutter.package_name }}/index.html [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{%- extends config.BASE_TEMPLATE %}
+
+{%- block page_body %}
+TODO: Example template, please remove if you do not need it.
```
Comment (jirikuncar) [inline invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/{{ cookiecutter.package_name }}/templates/{{ cookiecutter.package_name }}/index.html]: `{# ... #}`

File: invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/{{ cookiecutter.package_name }}/templates/{{ cookiecutter.package_name }}/index.html [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{{'{%- extends config.BASE_TEMPLATE %}'}}
+
+{{'{%- block page_body %}'}}
+TODO: Example template, please remove if you do not need it.
```
Comment (jirikuncar) [inline invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/{{ cookiecutter.package_name }}/templates/{{ cookiecutter.package_name }}/index.html]: please wrap in `{# ... #}`

File: invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/{{ cookiecutter.package_name }}/views.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    __name__,
+    template_folder='templates',
+    static_folder='static',
+)
+
+
+@blueprint.route("/")
+def index():
+    """Home Page."""
+    return render_template("{{ cookiecutter.package_name }}/index.html",
```
Comment (jirikuncar) [inline invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/{{ cookiecutter.package_name }}/views.py]: `s/"/'/`

File: invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/setup.py [hunk 6]
```diff
@@ -21,10 +22,21 @@
...
     entry_points={
         'console_scripts': [
-            '{{ cookiecutter.package_name }} = {{ cookiecutter.package_name }}.cli:cli'
+            '{{ cookiecutter.package_name }} = '
+            '{{ cookiecutter.package_name }}.cli:cli'
         ],
+        'invenio_base.blueprints': [
+            '{{ cookiecutter.package_name }} = '
+            '{{ cookiecutter.package_name }}.views:blueprint'
+        ]
```
Comment (jirikuncar) [inline invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/setup.py]: add `,`

File: invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/setup.py [hunk 7]
```diff
@@ -21,10 +22,21 @@
     platforms='any',
     entry_points={
         'console_scripts': [
-            '{{ cookiecutter.package_name }} = {{ cookiecutter.package_name }}.cli:cli'
+            '{{ cookiecutter.package_name }} = '
+            '{{ cookiecutter.package_name }}.cli:cli'
```
Comment (jirikuncar) [inline invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/setup.py]: add `,`

**Comment by hachreak** [comment]
ping @jirikuncar

File: invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/requirements-devel.txt [hunk 8]
```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#
```
Comment (jirikuncar) [inline invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/requirements-devel.txt]: we don't want to generate CERN license for new instances

File: invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/{{ cookiecutter.package_name }}/templates/{{ cookiecutter.package_name }}/index.html [hunk 9]
```diff
@@ -0,0 +1,10 @@
+{{'{%- extends config.BASE_TEMPLATE %}'}}
+
+{% raw %}
+{%- block page_body %}
+<div class="container">
+  TODO: Example homepage, please remove if you do not need it.
```
Comment (jirikuncar) [inline invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/{{ cookiecutter.package_name }}/templates/{{ cookiecutter.package_name }}/index.html]: please wrap it in `{# ... #}`

**Comment by hachreak** [comment]
ping @jirikuncar @tiborsimko

File: invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/requirements-devel.txt [hunk 10]
```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+-e git+https://github.com/inveniosoftware/invenio-marc21.git#egg=invenio-marc21
```
Comment (jirikuncar) [inline invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/requirements-devel.txt]: Please sort it. Thanks

**Comment by hachreak** [comment]
ping @jirikuncar

**Pull request merged**